packages/kbn-optimizer/limits.yml

```diff
@@ -110,7 +110,7 @@ pageLoadAssetSize:
   share: 71239
   snapshotRestore: 79032
   spaces: 57868
-  stackAlerts: 29684
+  stackAlerts: 58316
```

Comment:
Created a follow-up issue to reduce bundle size: https://github.com/elastic/kibana/issues/142902